draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1729,6 +1729,8 @@ RFC EDITOR: PLEASE REMOVE THIS SECTION.
 
 ## Version -13 to -14 ## {#sec-13-14}
 
+* Replaced CBOR simple value "null" with "nil".
```

Comment:
but why?

RFC 8949 does not mention "nil".
RFC 8610 has "nil" as a way to specify a CBOR null ("null" works too), but there is no CBOR simple value "nil".  The second paragraph of RFC 8610 may be  a bit misleading...

Reply:
That was in an old commit and the change was reverted :-) That changelog item is not present now in v -14, which only uses CBOR simple value "null".